config/schema/indexes/mainstream.json

```diff
@@ -16,6 +16,7 @@
     "medical_safety_alert",
     "policy",
     "raib_report",
-    "vehicle_recalls_and_faults_alert"
+    "vehicle_recalls_and_faults_alert",
+    "asylum_support_decision"
```

Comment:
Likewise this was added by generator using Rails' inject_into_file() method.  Placement control is limited to adding *before* or *after*.  Given this limitation it was only possible to identify and add to the end of the list.